app/commands/git/sync_doc.rb

```diff
@@ -16,14 +16,14 @@ def call
 
       doc.update!(attributes)
     rescue StandardError
-      # TODO: Raise issue on GH.
+      # TODO: (Blocking): Raise issue on GH.
```

Comment:
I think this might actually be something that we want to do. It's easy enough for me to do.